realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -3,6 +3,17 @@
 
 using namespace realsense2_camera;
 
+/* Modified for Brain */
+#include <stdio.h>
+
+static float unit_vectors[FRAME_MAX_HEIGHT][FRAME_MAX_WIDTH * 3]= {0};
```

Comment:
I think we should consider using std::vectors over C arrays. If we need to access the raw underlying array, we can always just use std::vector's `.data()` method.